README.md

```diff
@@ -1 +1,72 @@
-# capstone-1
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+Capstone 1.
+This is the first Microverse capstone project, in this project, we are to develop a website.
+The title of my website is Spytech Web Development Summit. It was design was inspired by Cindy Shin's design in Behance.
+
+Spytech Web Development Summit Website.
+
+Spytech Web development summit aims to establish a solid foundation in software development.
+
+## Built With
+
+- HTML and CSS and Javascript.
+## Technologies.
+Visual Studio Code.
+nodejs and npm.
+
+
+## Getting Started
+
+
+
+
+To get a local copy up and running follow these simple example steps.
+Locate the project folder in the Repository.
+Click on the code button and dowload a zip file for the project locally.
+Locate the index.html file and double click it to view webpage.
+
+### Prerequisites
+- Basic HTML & CSS knowledge
+- Basic Javasript knowledge
+Node.js, stylelint, browser for displaying it's content.
+install node package manager(npm): npm helps to install stylelint.
+
+
+### Setup
+git clone https://github.com/spydaspider/capstone-1.git
+
+### Usage
+Navigate through the web page to find the information you need. Use the links in the pages to move to different sections.
+
+
+### Deployment
+  Project reside in this Repo. You can clone the repo or download the files manually to your local machine.
+
+### Live Demo
+[Link to my Portfolio](https://spydaspider.github.io/microverse-projects/)
```

Comment:
- [ ] You should deploy your capstone project correctly and provide the link here instead of your portfolio link as in the requirements